pkg/resource/endpoint/testdata/v1alpha1/create/observed/success_after_create_deployment_config.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
+kind: Endpoint
+metadata:
+  creationTimestamp: null
+  name: xgboost-endpoint
+spec:
+  endpointConfigName: xgboost-endpoint-single-variant-config
+  endpointName: xgboost-endpoint
+  deploymentConfig:
+    autoRollbackConfiguration:
+      alarms:
+      - alarmName: 5xalarm-endpointtest
+    blueGreenUpdatePolicy:
+      maximumExecutionTimeoutInSeconds: 600
+      terminationWaitInSeconds: 100
+      trafficRoutingConfiguration:
+        linearStepSize:
+          type_: INSTANCE_COUNT
+          value: 1
+        type_: LINEAR
+        waitIntervalInSeconds: 100
+status:
```

Comment:
missing several fields

Reply:
This is right after the create call and before the subsequent readOne call, the controller wont know the status of the endpoint.